lib/ews/soap/exchange_availability.rb

```diff
@@ -27,7 +27,12 @@ def get_user_oof_settings(opts)
         }
         end
       end
-      do_soap_request(req, response_class: EwsSoapAvailabilityResponse)
+      options = {
+          request_type: 'Get User OOF Settings',
+          uniq_id: SecureRandom.uuid,
```

Comment:
why we are passing uuid from here rather than generate it from `#post`?

Reply:
good question, probs because I started with it here and had the login in dispatch then moved it to post, don't think it makes too much difference but will move it